internal/pkg/testhelpers/consensus.go

```diff
@@ -143,41 +140,45 @@ func (ms testSigner) SignBytes(data []byte, addr address.Address) (types.Signatu
 }
 
 // RequireActorIDAddress looks up an actor address in the init actor and returns the associated id address
-func RequireActorIDAddress(ctx context.Context, t *testing.T, st state.Tree, store vm.StorageMap, addr address.Address) address.Address {
-	processor := consensus.NewConfiguredProcessor(builtin.DefaultActors)
-	params, err := abi.ToEncodedValues(addr)
-	require.NoError(t, err)
+func RequireActorIDAddress(ctx context.Context, t *testing.T, st state.Tree, store vm.Storage, addr address.Address) address.Address {
+	// Dragons: why is this needed? delete
```

Comment:
Since actor code testing is out of scope of gfc with the new project I suspect most of these helpers can be deleted.